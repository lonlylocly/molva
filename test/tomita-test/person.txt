В Spiegel объяснили появление Меркель с нацистами на обложке журнала
RT @krabsnago: Шеф-редактор Spiegel объяснил появление на обложке окруженной нацистами
Канцлер Германии Ангела Меркель, президент Франции Франсуа Олланд и премьер-министр
Немецкие СМИ: пилоты авиакомпании Germanwings отказываются выполнять рейсы на самолетах A320
Вратарь сборной России Игорь Акинфеев, в которого попали файером в начале встречи
RT @suharig2015: Форум социальных работников в Ярославле, организованный партией "Единая Россия", стартует сегодня, 27 марта! http://t.co/I…
В Курске сильный ветер сдул со взлетной полосы самолет с пассажирами: В аэропорту Курска сильным порывом 
Канцлер
RT @Kevinsnydwr: Когда твоего друга нет в школе https://t.co/kTCq3XZJTe
