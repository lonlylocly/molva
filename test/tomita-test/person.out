В Spiegel объяснили появление Меркель с нацистами на обложке журнала 
	SimpleFact
	{
		Noun = Spiegel
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = объяснить
	}
	SimpleFact
	{
		Noun = появление
	}
	SimpleFact
	{
		Noun = Меркель
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = нацист
		Prep = с
	}
	SimpleFact
	{
		Noun = обложка
		Prep = на
	}
	SimpleFact
	{
		Noun = журнал
	}
RT @ krabsnago : Шеф-редактор Spiegel объяснил появление на обложке окруженной нацистами 
	SimpleFact
	{
		Noun = Шеф-редактор
	}
	SimpleFact
	{
		Noun = Spiegel
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = объяснить
	}
	SimpleFact
	{
		Noun = появление
	}
	SimpleFact
	{
		Noun = обложка
		Prep = на
	}
	SimpleFact
	{
		Noun = окруженный
	}
	SimpleFact
	{
		Noun = нацист
	}
Канцлер Германии Ангела Меркель , президент Франции Франсуа Олланд и премьер-министр 
	SimpleFact
	{
		Noun = канцлер
	}
	SimpleFact
	{
		Noun = Германий
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = Ангел
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = Меркель
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = президент
	}
	SimpleFact
	{
		Noun = Франций
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = Франсуа
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = Олланд
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = премьер-министр
	}
Немецкие СМИ : пилоты авиакомпании Germanwings отказываются выполнять рейсы на самолетах A320 
	SimpleFact
	{
		Noun = немецкий
	}
	SimpleFact
	{
		Noun = СМИ
	}
	SimpleFact
	{
		Noun = пилот
	}
	SimpleFact
	{
		Noun = авиакомпания
	}
	SimpleFact
	{
		Noun = Germanwings
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = отказываться
	}
	SimpleFact
	{
		Noun = рейс
	}
	SimpleFact
	{
		Noun = самолет
		Prep = на
	}
	SimpleFact
	{
		Noun = A320
		IsPersonName = true
	}
Вратарь сборной России Игорь Акинфеев , в которого попали файером в начале встречи 
	SimpleFact
	{
		Noun = вратарь
	}
	SimpleFact
	{
		Noun = сборный
	}
	SimpleFact
	{
		Noun = Россия
	}
	SimpleFact
	{
		Noun = Игорь
	}
	SimpleFact
	{
		Noun = Акинфеев
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = который
	}
	SimpleFact
	{
		Noun = файер
	}
	SimpleFact
	{
		Noun = начало
		Prep = в
	}
	SimpleFact
	{
		Noun = встреча
	}
RT @ suharig2015 : Форум социальных работников в Ярославле , организованный партией "Единая Россия" , стартует сегодня , 27 марта ! http : / / t.co / I … 
	SimpleFact
	{
		Noun = Форум
	}
	SimpleFact
	{
		Noun = социальный
	}
	SimpleFact
	{
		Noun = работник
	}
	SimpleFact
	{
		Noun = Ярославль
		Prep = в
	}
	SimpleFact
	{
		Noun = партия
	}
	SimpleFact
	{
		Noun = Единый
	}
	SimpleFact
	{
		Noun = Россия
	}
	SimpleFact
	{
		Noun = стартовать
	}
	SimpleFact
	{
		Noun = март
	}
В Курске сильный ветер сдул со взлетной полосы самолет с пассажирами : В аэропорту Курска сильным порывом 
	SimpleFact
	{
		Noun = Курск
		Prep = в
	}
	SimpleFact
	{
		Noun = сильный
	}
	SimpleFact
	{
		Noun = ветер
	}
	SimpleFact
	{
		Noun = сдуть
	}
	SimpleFact
	{
		Noun = взлетный
	}
	SimpleFact
	{
		Noun = полоса
	}
	SimpleFact
	{
		Noun = самолет
	}
	SimpleFact
	{
		Noun = пассажир
		Prep = с
	}
	SimpleFact
	{
		Noun = аэропорт
		Prep = В
	}
	SimpleFact
	{
		Noun = Курск
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = сильный
	}
	SimpleFact
	{
		Noun = порыв
	}
Канцлер 
	SimpleFact
	{
		Noun = канцлер
	}
RT @ Kevinsnydwr : Когда твоего друга нет в школе https : / / t.co / kTCq3XZJTe 
	SimpleFact
	{
		Noun = твой
	}
	SimpleFact
	{
		Noun = друг
	}
	SimpleFact
	{
		Noun = школа
		Prep = в
	}
Меркель , Олланд и Порошенко поставили условие : 
	SimpleFact
	{
		Noun = Олланд
		IsPersonName = true
	}
	SimpleFact
	{
		Noun = Порошенко
	}
	SimpleFact
	{
		Noun = поставить
	}
	SimpleFact
	{
		Noun = условие
	}
